### PR TITLE
Fix db dump collection

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #2502

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/utils.py [hunk 1]
```diff
@@ -831,12 +831,11 @@ def collect_ocs_logs(dir_name, ocp=True, ocs=True, mcg=False):
         )
     if mcg:
         counter = 0
-        mcg_db_collected = False
-        while not mcg_db_collected or counter < 5:
+        while counter < 5:
```
Comment (clacroix12) [inline ocs_ci/ocs/utils.py:834]: Why not use our [retry decorator](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/retry.py) for `collect_noobaa_db_dump`?

Comment (petr-balogh) [reply ocs_ci/ocs/utils.py:834]: This was done by @ebenahar , and I am just fixing the issue here. The retry will anyway the re-raise the exception after final retry. This is not the case here as we are just ignoring it here and continue without re-raising the exception.

**Comment by petr-balogh** [comment]
For sure I've triggered PR verification for tier1, but not sure if we will hit some failing test which will invoke must gather here.

**Comment by petr-balogh** [comment]
Only one dump done:
23:12:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig rsh noobaa-db-0 mongodump --archive=nbcore.gz --gzip --db=nbcore

**Pull request merged**